pkg/apis/workspaces/v1alpha2/component_compose.go

```diff
@@ -0,0 +1,38 @@
+package v1alpha2
+
+// ComposeFileComponentLocationType describes the type of
+// the location where the docker-compose file is fetched from.
```

Comment:
I would avoid mentioning docker here. You can see that's [the approach of in the official spec site too](https://compose-spec.io/).

```suggestion
// the location where the Compose file is fetched from.
```

Reply:
Yes makes sense, will remove comments mentioning docker.